Functions.Templates/Templates/SqlTrigger-CSharp/function.json

```diff
@@ -0,0 +1,13 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "name": "changes",
+      "type": "sqlTrigger",
+      "direction": "in",
```

Comment:
This seems to be missing the sql trigger values (tableName, connectionStringSetting)